src/assets/js/languageSelector.js

```diff
@@ -15,7 +15,13 @@ var languageSelector = {
     
     $('.section').hide();
     this.showSection(languageParam);
-    $('code').hide();
+    
+    $('code').each(function(){
+      if (typeof $(this).attr('class') != 'undefined' && $(this).attr('class') != "hljs") {
+        $(this).hide();
+      }
+    });
+    
     $('code.' + languageParam).show();
```

Comment:
Nitpicking: You can use string interpolation because we use babel to ensure compatibility

Reply:
Babel isn't used in our documentation build process

Reply:
Yes you're right, my bad :o